app/controllers/answers_controller.rb

```diff
@@ -39,6 +39,12 @@ def destroy
     @answer.destroy
   end
 
+  # GET /answers/question/1
+  def get_question
+    @answers = Answer.all.where(question_id: params["question_id"])
```

Comment:
```suggestion
    @answers = Answer.where(question_id: params["question_id"])
```

You shouldn't need the `all` here. Just by using the `where` it creates the AR query and when you try to use the `@answers` it'll run the query. `all` you use when you really want all.